### PR TITLE
Clean footer type styles 804

### PR DESCRIPTION
**Pull request by webuxr**
Fixes for #804.

### CODE REVIEW

File: _data/authors.yml [hunk 1]
```diff
@@ -47,6 +47,9 @@ nat_tarnoff:
 rachel_vasquez:
   name: Rachel R. Vasquez
   contact: http://rachievee.com
+ryan_r_smith:
```
Comment (ericwbailey) [inline _data/authors.yml]: Please remove this. `authors.yml` is metadata reserved for people who have written a [post](https://github.com/a11yproject/a11yproject.com/tree/gh-pages/_posts).

Comment (webuxr) [reply _data/authors.yml]: @ericwbailey Whoops... yep, I'll revert that change.

File: _layouts/default.html [hunk 2]
```diff
@@ -107,7 +107,7 @@
...
+                <path d="M-3116.4 353c-282 0-510.6 228.6-510.6 510.6s228.6 510.6 510.6 510.6s510.6-228.6 510.6-510.6S-2834.4 353-3116.4 353z M-3124.1 454.4c36.6 0 66.2 29.6 66.2 65.9c0 36.6-29.6 66.2-66.2 66.2c-36.3 0-65.9-29.6-65.9-66.2 C-3190.1 484-3160.5 454.4-3124.1 454.4L-3124.1 454.4z M-2838 667.7l-196.4 24.8l0.1 196.6l95.2 317.7c5 20.1-7 40.1-26.9 45.1 c-19.8 5-40.1-6-45.1-26l-97.5-289.2h-30l-89.7 295.2c-7.5 19.3-29 28.1-48.2 20.6c-19.1-7.4-30.7-29.1-23.2-48.4l82.4-311.8V692.5 l-181-24.6c-18.6-1.5-31.2-17.8-29.7-36.4c1.4-18.7 18.3-32.6 36.8-31.1l219.9 18.9h96.4l234.1-19.3c18.6-0.7 34.4 13.7 35 32.5 C-2805 651-2819.4 666.9-2838 667.7L-2838 667.7z">
```
Comment (webuxr) [inline _layouts/default.html]: FYI: Removed self-closing slash as it's not needed with this doctype.

File: _layouts/default.html [hunk 3]
```diff
@@ -92,7 +92,7 @@
   <body>
     <svg style="display: none;">
       <symbol id="icon-github" viewBox="0 0 32 32" fill="currentColor">
-        <path d="M0 18 C0 12 3 10 3 9 C2.5 7 2.5 4 3 3 C6 3 9 5 10 6 C12 5 14 5 16 5 C18 5 20 5 22 6 C23 5 26 3 29 3 C29.5 4 29.5 7 29 9 C29 10 32 12 32 18 C32 25 30 30 16 30 C2 30 0 25 0 18 M3 20 C3 24 4 28 16 28 C28 28 29 24 29 20 C29 16 28 14 16 14 C4 14 3 16 3 20 M8 21 A1.5 2.5 0 0 0 13 21 A1.5 2.5 0 0 0 8 21 M24 21 A1.5 2.5 0 0 0 19 21 A1.5 2.5 0 0 0 24 21 z "></path>
+        <path d="M0 18 C0 12 3 10 3 9 C2.5 7 2.5 4 3 3 C6 3 9 5 10 6 C12 5 14 5 16 5 C18 5 20 5 22 6 C23 5 26 3 29 3 C29.5 4 29.5 7 29 9 C29 10 32 12 32 18 C32 25 30 30 16 30 C2 30 0 25 0 18 M3 20 C3 24 4 28 16 28 C28 28 29 24 29 20 C29 16 28 14 16 14 C4 14 3 16 3 20 M8 21 A1.5 2.5 0 0 0 13 21 A1.5 2.5 0 0 0 8 21 M24 21 A1.5 2.5 0 0 0 19 21 A1.5 2.5 0 0 0 24 21 z"></path>
```
Comment (webuxr) [inline _layouts/default.html]: FYI: Removed extra space near end-of-line.

File: _layouts/default.html [hunk 4]
```diff
@@ -136,13 +136,14 @@ <h2 class="join-us-title">Join us on <a href="https://github.com/a11yproject/a11
...
-          <ul id="contributors-list" class="row"></ul>
+          <ul class="contributors-list"></ul>
           <ul class="pager contributors-pagination inline-list"></ul>
         </div>
 
         <div class="colophon">
-
           <h5 class="screen-reader-text">Colophon</h5>
+
+          <p class="no-lead">Stay in touch</p>
```
Comment (webuxr) [inline _layouts/default.html]: In terms of the phrase I used here... does anyone have thoughts on adding this bit of text (of if it should say something else)?

**Comment by scottaohara** [comment]
@webuxr could we please keep the PR in scope to its intent?  e.g. there should be no reason to update the gems, or merging in of other branches.

we're looking to do away with the gems in general, so this would just cause us some conflicts for other pending work.  

Thanks

File: _layouts/default.html (hunk 4 above)
Comment (scottaohara) [reply _layouts/default.html]: this is fine.  let's not add anymore beyond this though.

**Comment by webuxr** [comment]
@scottaohara I'm not sure how/why [this commit from 2017](https://github.com/a11yproject/a11yproject.com/pull/806/commits/7afee9b71007ff8ffb9c13b7a8d4ec60a6a6e1e0) made it into my PR, but I think it's best I start over with my PR to ensure I'm branching from the tip of gh-pages, squash my commits, etc. Plus, I'll ensure my changes map directly to issue #804.

**Pull request closed without merging**